crates/sui-graphql-rpc/src/context_data/module_resolver.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use diesel::ExpressionMethods;
+use diesel::QueryDsl;
+use diesel::RunQueryDsl;
+use move_core_types::language_storage::ModuleId;
+use move_core_types::resolver::ModuleResolver;
+use sui_indexer::schema_v2::packages;
+
+use sui_indexer::models_v2::packages::StoredPackage;
+use sui_sdk::types::base_types::ObjectID;
+use sui_sdk::types::move_package::MovePackage;
+
+use super::db_data_provider::diesel_macro::read_only_blocking;
+use super::db_data_provider::PgConnectionPool;
+
+use crate::error::Error;
+
+pub(crate) struct PgModuleResolver {
```

Comment:
the implementation looks the same as IndexerStoreModuleResolver in module_resolver_v2.rs? Why not directly use it? Are we afraid of unnecessary crate dependencies? I see it's already using stuff like `StoredPackage`

Reply:
Maybe I'm just stupid but I couldn't import it for whatever reason. I'll try again

Reply:
I gotta clean this PR up haha, still a draft but undrafted it for visibility

Reply:
Looks like IndexerStoreModuleResolver is only public to the crate. Any concerns if I make it public?